packages/pf4/site/src/components/navigation/navigation.styles.js

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet } from '@patternfly/react-styles';
+
+export default StyleSheet.create({
+  navigation: {
+    width: 300
+  },
+  navigationContent: {
+    position: 'fixed',
+    width: 'inherit',
+    height: '100vh',
+    backgroundColor: '#fff'
```

Comment:
Can we use color variables here?

Reply:
Will do.  I found a pretty good way around hard coding these that matches how salesforce and firefox are doing something similar.  Will add.

Reply:
These have been updated to use a new package `@patternfly/react-tokens` this will allow the values to be accessed directly @priley86 as well as the name from JavaScript.

Reply:
wow - terrific!!! I think these will definitely come in handy when writing extensions and become quite helpful for us.

I think this PR looks great...

can you add `@patternfly/react-tokens` to your `affects` clause?